common/src/main/java/com/microsoft/identity/common/internal/cache/MsalOAuth2TokenCache.java

```diff
@@ -672,6 +672,27 @@ private Credential getFamilyRefreshTokenForAccount(@NonNull final AccountRecord
         return result;
     }
 
+    /**
+     * Load  FRTs from the cache for an account matching the homeAccountId
+     * @param homeAccountId : homeAccountId for which FRT is sought
+     * @return an FRT if available else null.
+     */
+    @Nullable
+    public RefreshTokenRecord getFamilyRefreshTokenForHomeAccountId(@NonNull final String homeAccountId) {
+        final List<AccountRecord> accountRecords = mAccountCredentialCache.getAccounts();
+
+        for (AccountRecord accountRecord : accountRecords) {
+            if (accountRecord.getHomeAccountId().equals(homeAccountId)) {
+                final Credential rt = getFamilyRefreshTokenForAccount(accountRecord);
+
+                if (rt instanceof RefreshTokenRecord) {
+                    return (RefreshTokenRecord) rt;
+                }
```

Comment:
When does this return something that's not a RefreshTokenRecord?  If I go look at the code, it only sets credential being returned if it is instanceof RefreshTokenRecord, I think.  It's a private method - should we change the signature?

Reply:
Good point, agree. Addressed via https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/920/commits/fff3f7e02530a7052cbd302dc428616fff91c435